mkosi/__init__.py

```diff
@@ -3951,12 +3951,16 @@ def make_verity_sig(
     from cryptography import x509
     from cryptography.hazmat.primitives import hashes, serialization
     from cryptography.hazmat.primitives.serialization import pkcs7
+    from cryptography.hazmat.primitives.asymmetric import ec, rsa
```

Comment:
isort wants this line one line up, I think, since `asymmetric` comes before `serialization`.